src/components/LeftPanel/LeftPanel.tsx

```diff
@@ -41,10 +41,12 @@ export default function LeftPanel() {
 			<Grid xs item>
 				<Box mb={1} mt={1}>
 					<IconButton>
-						<Avatar
-							variant="square"
-							src={require("../../assets/icons/Fundimpact-logo.png")}
-						/>
+						<Link href="/organization/dashboard" color="inherit" underline="none">
```

Comment:
Using href will load the entire page . Use Link from `react-router-dom` with `to` prop

Reply:
Ohky